jerry-core/jerryscript.h

```diff
@@ -389,6 +389,11 @@ size_t jerry_parse_and_save_literals (const jerry_char_t *source_p, size_t sourc
                                       uint32_t *buffer_p, size_t buffer_size, bool is_c_format);
 
 /**
+ * External context functions.
+ */
+size_t jerry_get_context_size (void);
+size_t jerry_get_lcache_size (size_t row_count, size_t row_length);
```

Comment:
This will be public API? I don't expect a user to understand what is lcache, and I would like to reserve the right to change such internal stuff (e.g. replace lcache with something else). So we definitely need something better here.